components/brave_rewards/browser/net/network_delegate_helper.cc

```diff
@@ -75,19 +75,17 @@ void DispatchOnUI(
 int OnBeforeURLRequest(
   const brave::ResponseCallback& next_callback,
   std::shared_ptr<brave::BraveRequestInfo> ctx) {
-  DCHECK_CURRENTLY_ON(content::BrowserThread::IO);
+  DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
```

Comment:
I see that this was changed, but cant' find where we actually change it from `IO` to `UI` so that this check would now pass

Reply:
in `BraveRequestHandler`

Reply:
thx